internal/daysteps/daysteps.go

```diff
@@ -13,8 +20,51 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	err1 := errors.New("data format error")
+	err2 := errors.New("error of conversion step in pP")
+	err3 := errors.New("stpcount not recognized")
+	err4 := errors.New("error of conversion duration in pP")
+	err5 := errors.New("durcount not recognized")
+	dataslise := strings.Split(data, ",")
+	if len(dataslise) != 2 {
+		return 0, 0, err1
```

Comment:
Проще на месте делать ошибки, так более читаемо
```suggestion
		return 0, 0, fmt.Errorf("length of data is not 2")
```